src/DIRAC/TransformationSystem/Client/TransformationCLI.py

```diff
@@ -1,13 +1,8 @@
 """ Transformation Database Client Command Line Interface.
 """
+import DIRAC
 
-from __future__ import print_function
-from __future__ import absolute_import
-from __future__ import division
-
-from DIRAC.Core.Base.Script import parseCommandLine
-
-parseCommandLine()
+DIRAC.initialize()  # Initialize configuration
```

Comment:
I realise this was a pre-existing issue but can this be refactored to make it safe to import?